source/plugins/ruby/CustomMetricsUtils.rb

```diff
@@ -13,6 +13,11 @@ def check_custom_metrics_availability
             if aks_region.to_s.empty? || aks_resource_id.to_s.empty?
                 return false # This will also take care of AKS-Engine Scenario. AKS_REGION/AKS_RESOURCE_ID is not set for AKS-Engine. Only ACS_RESOURCE_NAME is set
             end
+            # If this is cluster is connected to ArcA control plane and metrics endpoint provided, custom metrics shall be emitted.
+            is_ArcA_Cluster = ENV['IS_ARCA_CLUSTER']
+            if is_ArcA_Cluster.to_s.downcase == "true" && !ENV['Custom_Metrics_Endpoint'].to_s.empty?
```

Comment:
In case of Azure, custom metrics are only available in public cloud. what regions and clouds (if there is any) supported in case of Arc-A?  is that assumption, as long as custom metrics endpoint provided custom metrics available in case of Arc-A?

Reply:
1. In Arc-A, the regions could be different since the cluster could placed on different areas. Also, the clouds (found from domain) is not usable for us, since the domain of Arc-A is a FQDN from user internal network. So currently we have to add a parameter IS_ARCA_CLUSTER to mark Arc-A cluster rather than clouds.
2. Yep, the assumption is right. Since the metrics endpoint is provided by cluster admin, and custom metrics is supported from Arc-A preview 1, it can be confirmed to provide custom metrics in this case.